client/extensions/woocommerce/app/store-stats/referrers/chart/index.js

```diff
@@ -0,0 +1,81 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import StoreStatsChart from 'woocommerce/app/store-stats/store-stats-chart';
+import Tabs from 'my-sites/stats/stats-tabs';
+import Tab from 'my-sites/stats/stats-tabs/tab';
+import { formatValue } from 'woocommerce/app/store-stats/utils';
+import { referrerChartTabs as tabs } from 'woocommerce/app/store-stats/constants';
+import { getStoreReferrersByReferrer } from 'state/selectors';
+
+class Chart extends Component {
+	static propTypes = {
+		data: PropTypes.array.isRequired,
+		selectedDate: PropTypes.string,
+		selectedReferrer: PropTypes.string,
+		chartFormat: PropTypes.string,
+	};
+
+	formatTabValue = ( tab, item ) => {
+		return value => formatValue( value, tab.type, item.currency );
+	};
+
+	renderTabs = ( { chartData, selectedIndex, selectedTabIndex, tabClick } ) => {
```

Comment:
Curious as to why `renderTabs` is here and not in `StoreStatsChart`?

Reply:
Good question. The orders chart has tabs that include `delta` widgets and are constructed differently than the referrer tabs. 

I could move both `renderTabs` functions to `StoreStatsChart` and conditionally apply. Would that be more clear?